### DISCUSSION TITLE
Rotate table transformation - to use table data for Graph, Bar gauge and etc

### DISCUSSION THREAD

**Discussion by zs-dima** [grafana/grafana#38141] [Dashboards, Panels & Visualizations]
Could be nice to have transformation to rotate table in a way:
from:
```
-----------------------
ColumnA  | ColumnB
-----------------------
A1       | B1
A2       | B2
A3       | B3
```
to:
```
----------------
A1  |  A2  |  A3
----------------
B1  |  B2  |  B3
```

I had not found any way to do it in case of using JSON Data Source plugin for tables to use table data for Graph, Bar gauge and etc

Thanks in advance

**Comment by torkelo** [comment]
Like a pivot?

**Comment by torkelo** [comment]
related to discussions in https://github.com/grafana/grafana/issues/15279

**Comment by torkelo** [comment]
and also related to https://github.com/grafana/grafana/issues/14814

**Comment by zs-dima** [comment]
> 
> 
> Like a pivot?

yes, data transformation - to use table data for Graph, Bar gauge and etc
and it is no 'group by' for the JSON Data Source plugin data table
thanks for the fast reply

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
